### PR TITLE
Changed the homepage slogan

### PR DESCRIPTION
**Pull request by francisli**
What I did here... changed the homepage text to demo git branch and team collaboration on Github.

### CODE REVIEW

File: client/src/Home.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function Home() {
         <h1>
           Welcome <span className="d-none d-md-inline">to SF Dreamer</span>
         </h1>
-        <h2>Welcome to SF Dreamer! Follow the Steps below to learn more about the Dream Act and figure out what to do.</h2>
+        <h2>This is the Francis Demo.</h2>
```
Comment (francisli) [inline client/src/Home.js:10]: This is a terrible slogan... please change it.

**Review by francisli — COMMENTED** [review]
Finish this review...

**Review by GaelGil — COMMENTED** [review]
Looks good

File: client/src/Home.js (hunk 1 above)
Comment (alex19992020) [reply client/src/Home.js]: I like it

Comment (JesusEArambula) [inline client/src/Home.js]: Hey that's something we should plan on for the future! :)

File: client/src/Home.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ function Home() {
         <h1>
           Welcome <span className="d-none d-md-inline">to SF Dreamer</span>
         </h1>
-        <h2>Welcome to SF Dreamer! Follow the Steps below to learn more about the Dream Act and figure out what to do.</h2>
+        <h2>SFDreamer is amazing!</h2>
```
Comment (GaelGil) [inline client/src/Home.js:10]: This is a good slogan

**Review by haikelfd — APPROVED** [review]
Looks Great

File: client/src/Home.js (hunk 2 above)
Comment (alex19992020) [inline client/src/Home.js:10]: I like this one!

**Review by JesusEArambula — APPROVED** [review]

**Pull request merged**